{{ cookiecutter.__project_name_kebab_case }}/README.md

```diff
@@ -1,19 +1,47 @@
-[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url={{ cookiecutter.package_url.replace("https://", "git@").replace(".com/", ".com:") if cookiecutter.private_package_repository_url else cookiecutter.package_url }})
+[![Open in Dev Containers](https://img.shields.io/static/v1?label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url={{ cookiecutter.project_url.replace("https://", "git@").replace(".com/", ".com:") if cookiecutter.private_package_repository_url else cookiecutter.project_url }})
 
-# {{ cookiecutter.package_name }}
+# {{ cookiecutter.project_name }}
 
-{{ cookiecutter.package_description }}
+{{ cookiecutter.project_description }}
+{%- if cookiecutter.project_type == "package" or cookiecutter.with_typer_cli|int %}
 
-## Using
+## Installing
+
+To install this package, run:
```

Comment:
Good catch, will improve.

Reply:
Solved by conditionally making it `poetry add` instead of `pip install` (Poetry is already configured for private package repos if the user selected that feature).

Reply:
I would expect this to be instructions for the package users not developers, in which case poetry isn't necessarily configured, if they even use poetry.

Reply:
I imagine that if you’re using a private package repo, that the repo itself will be private and therefore its contributors will be its users. And contributors will have configured Poetry correctly already. Do you have a different use case in mind?

Reply:
I was imagining a case where one team is developing a python packages to be consumed by other teams within the same organisation. Those other teams don't necessarily also contribute to the package.

Reply:
But if they’re in the same organisation, I think it would be reasonable to assume that consumers of the private package repo would also be using Poetry as that is what the team uses who maintain the package?

Reply:
Discussed with @JWuzyk and we concluded that we'll resolve the comment for now, but keep track of it with an issue that asks to explain how to add credentials for different package management tools.